### ISSUE TITLE
[airflow] ImportError from psycopg in Airflow chart (v. 1.10.3)

### ISSUE THREAD

**Issue by ghost** [helm/charts#15023]
**Describe the bug**

Following the default setup instructions [here](https://hub.helm.sh/charts/stable/airflow), I encounter dependency issues when running the Airflow system in Kubernetes. On first glance it appears to be a missing dependency or broken path for psycopg.

**Version of Helm and Kubernetes**:

I'm running an EKS 1.13 cluster.

Kubernetes version:

```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:11:31Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.6-eks-d69f1b", GitCommit:"d69f1bf3669bf00b7f4a758e978e0e7a1e3a68f7", GitTreeState:"clean", BuildDate:"2019-02-28T20:26:10Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```

Helm version:

```
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.1", GitCommit:"5270352a09c7e8b6e8c9593002a73535276507c0", GitTreeState:"clean"}
```

**Airflow chart version: 1.10.3**

**Which chart**:

Airflow

**What happened**:

The flower, scheduler, web, and worker pods all throw the same error, presumably while talking to Postgres:

```
[2019-06-24 17:29:21,138] {{settings.py:174}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=1
Traceback (most recent call last):
  File "/usr/local/bin/airflow", line 21, in <module>
    from airflow import configuration
  File "/usr/local/lib/python3.6/site-packages/airflow/__init__.py", line 36, in <module>
    from airflow import settings, configuration as conf
  File "/usr/local/lib/python3.6/site-packages/airflow/settings.py", line 266, in <module>
    configure_orm()
  File "/usr/local/lib/python3.6/site-packages/airflow/settings.py", line 188, in configure_orm
    engine = create_engine(SQL_ALCHEMY_CONN, **engine_args)
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/__init__.py", line 443, in create_engine
    return strategy.create(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 87, in create
    dbapi = dialect_cls.dbapi(**dbapi_args)
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 599, in dbapi
    import psycopg2
  File "/usr/local/lib/python3.6/site-packages/psycopg2/__init__.py", line 50, in <module>
    from psycopg2._psycopg import (                     # noqa
ImportError: libpq.so.5: cannot open shared object file: No such file or directory
```

These pods get into `CrashLoopBackOff` status and don't recover.

**What you expected to happen**:

Airflow system should start up without any exceptions.

**How to reproduce it** (as minimally and precisely as possible):

Using the Kubernetes version described above, create a namespace "airflow" and run the following (taken from the setup instructions):

`helm install --namespace "airflow" --name "airflow" stable/airflow`

**Anything else we need to know**:

**Comment by stevenmanton** [comment]
This is an issue with the underlying Docker image, but I thought that they had fixed it with the [`1.10.3`](https://hub.docker.com/r/puckel/docker-airflow/tags) tag. See #14847.

**Comment by stevenmanton** [comment]
Also, see this thread: https://github.com/puckel/docker-airflow/issues/342
